libs/barista-components/stacked-series-chart/README.md

```diff
@@ -145,11 +146,11 @@ This `DtStackedSeriesChartLegend` holds the information for every legend item so
 color and visibility is unified among same chart but also distributed charts
 (i.e. multiple charts in a table).
 
-| Name      | Type                | Description                                |
-| --------- | ------------------- | ------------------------------------------ |
-| `label`   | `string`            | Label of the node.                         |
-| `color`   | `DtColors | string` | Color to be used based on nodes and theme. |
-| `visible` | `boolean`           | Whether it should be visible.              |
+| Name      | Type      | Description                   |
+| --------- | --------- | ----------------------------- |
+| `label`   | `string`  | Label of the node.            |
+| `color`   | `DtColors | string`                       | Color to be used based on nodes and theme. |
```

Comment:
Something weird happened with the formatting. Have you checked it's correct in the docs?